django/thunderstore/repository/templates/repository/package_docs.html

```diff
@@ -77,6 +88,42 @@
                     <td>URL of the mod's website (e.g. GitHub repo). Can be an empty string.</td>
                     <td><pre>"https://example.com/"</pre></td>
                 </tr>
+                <tr>
+                    <td>installers</td>
+                    <td>❌</td>
+                    <td>
+                        <p>
+                            A list of installer declarations. Installer declarations can be used to explicitly
+                            control how a mod manager should install the package. If omitted, legacy install
+                            rules are automatically used.
+                        </p>
+                        <p>
+                            As of January 2024, the mod managers don't yet use this field for anything. Documentation
+                            will be updated with more details once an implementation exists.
+                        </p>
+                        <p>
+                            Documentation for the default (legacy) behavior is currently maintained as a wiki page on
+                            <a href="https://github.com/ebkr/r2modmanPlus/wiki/Structuring-your-Thunderstore-package">
+                                the r2modmanPlus wiki
+                            </a>
+                        </p>
+                        <p>
+                            This field should either contain a list of at least one valid installer declarations
```

Comment:
Gut feeling says it should be "list of at least one valid installer declaration" without the plural s but I'm not sure.